lib/src/migrators/module.dart

```diff
@@ -352,11 +356,11 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     if (declaration.isForwarded) return;
 
     var name = declaration.name;
-    if (name.startsWith('-') &&
+    if (_isPrivate(name) &&
         references.referencedOutsideDeclaringStylesheet(declaration)) {
-      // Remove leading `-` since private members can't be accessed outside
-      // the module they're declared in.
-      name = name.substring(1);
+      // Private members can't be accessed outside the module they're declared
+      // in.
+      name = _privateToPublic(name);
     }
     name = _unprefix(name);
```

Comment:
I believe the unprefixing should be done before the       `name = _privateToPublic(name);` to avoid the other issue I outlined

I am not sure what would cause the other issue but I believe a conflicting patch